Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM python:3.10-slim-buster
+
+WORKDIR /code
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        gcc \
+        postgresql-client \
+        build-essential \
+        nodejs \
+        npm \
+        libpq-dev && \
+    rm -rf /var/lib/apt/lists/*
+
+COPY requirements.txt /code/
+RUN pip install --no-cache-dir -r requirements.txt
+
+# Install requirements for running tests
+COPY ./tools/requirements-test.txt /code/
```

Comment:
keeping test requirements in is not a good practice these should be installed while running on CI

Reply:
Good point, I'll raise a fix over the weekend. Thanks.

Reply:
Let's keep this for now and move ahead. We will fix this while working on CI.